.changelog/3769.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:feature
+core: users can now view details about historical pipeline runs, and see details of specific pipeline runs
```

Comment:
Small note - I haven't added the changelog entry for pipelines yet but I think we could have it be all included in a single entry when we do, thoughts? 🤔